### PR TITLE
Unit tests update

### LINKED ISSUE #230: Research Error Handling process and Update Unit tests
**Is this related to a problem? Please describe.**

1) Investigate current error handling process
- Find issues 
- Remove clutter

2) Research a ways to improve error handling process

Hopefully next week we can re-factor our error handling process to match our current code base

### PR DESCRIPTION
**Pull request by edouarde1**
We have unit tests for phoneme processor. 

Add some TODO for updating error handling updates (future issue)

### CODE REVIEW

File: speaking-portal-project/src/types.ts [hunk 1]
```diff
@@ -9,8 +9,6 @@ export type MouthCue = {
     value: Phoneme
 }
 
-export type MouthCueArray = Array<MouthCue>
```
Comment (LoganParker) [inline speaking-portal-project/src/types.ts:12]: I like this change :)

**Review by LoganParker — APPROVED** [review]
Overall great changes! I like that we're really getting deep into the testing, I think it's an important thing to really touch on to make sure everything works as we expect

File: speaking-portal-project/src/types.ts (hunk 1 above)
Comment (vnajack) [reply speaking-portal-project/src/types.ts:12]: me too!

**Review by vnajack — APPROVED** [review]
Thank you so much for keeping the testing updated. It's super helpful and it's a great learning experience!

File: speaking-portal-project/src/types.ts (hunk 1 above)
Comment (mkuelker) [reply speaking-portal-project/src/types.ts:12]: me three!

**Review by mkuelker — APPROVED** [review]
Looks good

**Pull request merged**